File: examples-sdk/csharp/303-rag-hybrid-exact-semantic/Main.cs
// 303-rag-hybrid-exact-semantic — C# SDK equivalent
// Compile: vil compile --from csharp --input 303-rag-hybrid-exact-semantic/Main.cs --release

#r "sdk/csharp/Vil.cs"

var server = new VilServer("rag-hybrid-exact-semantic", 3112);
var rag_hybrid = new ServiceProcess("rag-hybrid");
rag_hybrid.Endpoint("POST", "/hybrid", "hybrid_handler");
server.Service(rag_hybrid);
server.Compile();

File: examples-sdk/kotlin/303-rag-hybrid-exact-semantic/main.kt
// 303-rag-hybrid-exact-semantic — Kotlin SDK equivalent
// Compile: vil compile --from kotlin --input 303-rag-hybrid-exact-semantic/main.kt --release

fun main() {
    val server = VilServer("rag-hybrid-exact-semantic", 3112)
    val rag_hybrid = ServiceProcess("rag-hybrid")
    rag_hybrid.endpoint("POST", "/hybrid", "hybrid_handler")
    server.service(rag_hybrid)
    server.compile()
}
